README

```diff
@@ -0,0 +1,13 @@
+# O3DE ROS2 Gem
+
+This Gem enables users to develop robotic simulations through ROS2 tools and components.
+
+# Adding Gem to your project
+
+To use this Gem in your project, you need to register the Gem with O3DE. First, clone this repository. Then,
+in o3de folder:
+```
+scripts/o3de.sh register --gem-path <PATH_TO_CLONED_ROS2_GEM>
+scripts/o3de.sh enable-gem -gn ROS2 -pp <PATH_TO_YOUR_PROJECT>
+```
```

Comment:
You may want to give this file the extension of .md so that it text applications pick it up as markdown by default

Reply:
I cherry-picked the commit, it was the very next on development branch and indeed belongs here